src/mibase.ts

```diff
@@ -432,7 +432,11 @@ export class MI2DebugSession extends DebugSession {
 
 	protected override async variablesRequest(response: DebugProtocol.VariablesResponse, args: DebugProtocol.VariablesArguments): Promise<void> {
 		const variables: DebugProtocol.Variable[] = [];
-		const id: VariableScope | string | VariableObject | ExtendedVariable = this.variableHandles.get(args.variablesReference);
+		let id: VariableScope | string | VariableObject | ExtendedVariable = this.variableHandles.get(args.variablesReference);
+
+		if(typeof id == "string" && id.includes("static ")) {
+			id = id.replace("static ", "");
+		}
```

Comment:
please move that to the place where that `id` with type string is actually used in this function, this both the review and following the code later an easier task

Reply:
thanks, i moved this handle to  'typeof id == "string" ' branch

Reply:
Ah, I see. I suggest to just use `const string name = id.replace(/^static /, "");` (replace also checks for the include) + using name instead of id in the two places below - this way you can also leave the "id" definition as is.
I'll pull that afterwards, so we can go to the next PR.

Thanks for your work on this!